### ISSUE TITLE
Deleted party

### ISSUE THREAD

**Issue by ABZB** [trigger-death/TriggersPC#5]
This is possibly outside the intended scope of the program, but when a save from an Emerald that was modified to have more Pokemon was loaded, all Pokemon in the party were deleted (the only 'new' Pokemon in that save were in the party).

(The particular mod overwrote some of the Glitch Pokemon slots (between Celebi and Treecko) with those Pokemon)

**Comment by trigger-death** [comment]
You are correct, unfortunately this is outside of the scope of Trigger's PC. supporting modded games becomes difficult when the structure can vary so much.

Could you leave the name of the game/mod in the title so that others can take note?

**Comment by ABZB** [comment]
Sure.

This was not a specific mod - I had modded a stock Emerald to insert Pokemon based on the some of the Delta Pokemon from the TCG in the slots between Celebi and Treecko (index numbers 252 and 254 in this case specifically).

I suspect similar issues might arise with mods using any of the empty item indices.

**Comment by trigger-death** [comment]
Yeah, it's definitely a guarantee to expect non-existent IDs to produce undefined behavior. I haven't really worked with the codebase since 2016 so I can't recall exactly how it handles things. But my guess would be that it eliminates any undefineds.
